nupic/embodied/envs/real_robots/wrappers.py

```diff
@@ -19,12 +19,11 @@
 #  http://numenta.org/licenses/
 #
 # ------------------------------------------------------------------------------
-import gym
 from gym import spaces
-import numpy as np
+
 
 class WrapRobot(object):
-    '''
+    """
```

Comment:
Use only double quotes, except when already inside double quotes (then single quotes would be required).